src/contracts/Yieldy.sol

```diff
@@ -241,4 +227,50 @@ contract Yieldy is
     function decimals() public view override returns (uint8) {
         return decimal;
     }
+
+    /**
+        @notice called from the staking contract co create Yieldy tokens
+        @param _address to receive tokens
+        @param _amount to mint to _address
+     */
+    function mint(address _address, uint256 _amount)
+        external
```

Comment:
should override?

Reply:
checking, but I am not sure what it would override in the dependency chain.

Reply:
Yea, nothing there to override 


```
TypeError: Function has override specified but does not override anything.
   --> src/contracts/Yieldy.sol:257:54:
    |
257 |     function burn(address _address, uint256 _amount) override
    |                                                      ^^^^^^^^
```

Reply:
it would need to override _mint

function _mint(address account, uint256 amount) override external {

We already have a _mint that's internal, if that gets called somehow, then it may not mint correctly

Reply:
ah, you are referring to the internal _mint and _burn in the ERC20 we inherit from OZ.  While I am not sure how they would ever get called given the current implementation, sure we can override them.

Reply:
@toshiSat - added.

Reply:
yea, someone would have to explicitly write code, but thanks for adding that!